tmt/libraries/beakerlib.py

```diff
@@ -276,8 +275,7 @@ def fetch(self) -> None:
         except (KeyError, FileNotFoundError):
             self.parent.debug(f"Fetch library '{self}'.", level=3)
             # Prepare path, clone the repository, checkout ref
-            assert self.parent.workdir
-            directory = self.parent.workdir / self.dest / self.repo
+            directory = self.step_workdir / self.dest / self.repo
```

Comment:
Similar here.

Reply:
Removed in c5b1be5a802b35f19e03939d9ebfa52a392caa60